Sources/Conformance/conformance.pb.swift

```diff
@@ -491,6 +491,7 @@ extension Conformance_ConformanceRequest: SwiftProtobuf.Message, SwiftProtobuf._
       switch fieldNumber {
       case 1: try {
         var v: Data?
+        if case .protobufPayload(let _v) = self.payload {v = _v}
         try decoder.decodeSingularBytesField(value: &v)
         if let v = v {
           if self.payload != nil {try decoder.handleConflictingOneOf()}
```

Comment:
This part I'm also no longer sure of.

If the oneof was already set to the same failed, is it always a potentially conflicting oneof? It likely is up to the decoder, but there is no longer enough info to say if it was already set to the same value or was set to a different oneof value.